pyproject.toml

```diff
@@ -32,6 +32,7 @@ test = ["pytest"]
 
 [project.scripts]
 pyspdxtools = "spdx.clitools.pyspdxtools:main"
+pyspdxtools3 = "spdx3.clitools.pyspdxtools3:main"
```

Comment:
Yes, I also thought about this. But as this would possibly require another restructuring (put `clitools.py` into `common`),  I decided to leave this for later.